### ISSUE TITLE
Making GUI layout more flexible

### ISSUE THREAD

**Issue by AlexEMG** [DeepLabCut/DeepLabCut#110]
In the current version the dimensions of the GUIs (including where the buttons are etc.) are hard coded. It would be wonderful if someone would like to make this more flexible (especially for smaller screens).  Thanks!

**Comment by MMathisLab** [comment]
Values are in here: https://github.com/AlexEMG/DeepLabCut/blob/master/deeplabcut/generate_training_dataset/labeling_toolbox.py

**Comment by MMathisLab** [comment]
if someone wants to try an update, you can use this repo: https://github.com/MMathisLab/DeepLabCut - it would be great if a Windows user would test it....( maybe @banningjoe? :)

**Comment by AlexEMG** [comment]
Fixed in commit https://github.com/AlexEMG/DeepLabCut/commit/aafbeb2db472d8450cde14886eb3bc40e967ccaf

**Comment by emdbell17** [comment]
I have a problem with the "analyze videos" part of the GUI. When I try to plot trajectories the screen maxes out and I can't scroll to hit run. I found this thread to try and figure out what dimensions deeplabcut uses, but they are no longer available. The link gives a 404 not found error. If someone could point me to these dimensions or suggest any solutions to be able to use the GUI better that would be awesome! I also tried using a monitor with a larger screen than my mac but it's still not the length deeplabcut is using!
